linera-chain/src/unit_tests/chain_tests.rs

```diff
@@ -171,11 +171,11 @@ async fn test_block_size_limit() {
 
     // The valid block is accepted...
     let outcome = chain.execute_block(&valid_block, time, None).await.unwrap();
-    let executed_block = outcome.with(valid_block);
+    let block: Block = Block::new(valid_block, outcome);
```

Comment:
```suggestion
    let block = Block::new(valid_block, outcome);
```

(Or `outocome.with(valid_block)`? You still use that below.)

Reply:
`outcome-.with(valid_block)` still returns `ExecutedBlock` while here we want `Block`.

Reply:
Done in [Address review comments from Andreas](https://github.com/linera-io/linera-protocol/pull/3101/commits/89062f6fb4b18b5d5a3b037719617210e82123e4)